klog.go

```diff
@@ -54,11 +56,23 @@ func (v Verbose) Infof(format string, args ...interface{}) {
 	}
 }
 
+func (v Verbose) InfoS(msg string, keysAndValues ...interface{}) {
+	if v {
+		level.Debug(logger).Log("func", "Verbose.InfoS", "msg", msg, keysAndValues)
+	}
+}
+
+func (v Verbose) InfoSDepth(_ int, msg string, keysAndValues ...interface{}) {
+	if v {
+		level.Debug(logger).Log("func", "Verbose.InfoSDepth", "msg", msg, keysAndValues)
+	}
+}
+
```

Comment:
Do these new functions need to be added to the table in the README?

Reply:
nice catch!